lib/mongodb.js

```diff
@@ -483,6 +483,25 @@ MongoDB.prototype.updateOrCreate = function updateOrCreate(model, data, options,
 };
 
 /**
+ * Replace if the model instance exists with the same id or create a new instance
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model instance data
+ * @param {Function} [cb] The callback function
+ */
+MongoDB.prototype.replaceOrCreate = function (model, data, options, cb) {
+  var self = this;
```

Comment:
Is `var self = this` even need? I don't see any case in this function that requires it.